luda-editor/client/src/pages/sequence_edit_page/loaded/components/image_manager_modal/update/mod.rs

```diff
@@ -8,6 +9,22 @@ impl ImageManagerModal {
                     namui::event::send(Event::Error(error.clone()));
                 }
             }
+        } else if let Some(event) = event.downcast_ref::<InternalEvent>() {
+            match event {
+                InternalEvent::RequestUploadImages => {
+                    let project_id = self.project_id;
+                    spawn_local(async move {
+                        let result = upload_images(project_id).await;
+                        match result {
+                            Ok(_) => namui::event::send(InternalEvent::UploadImageFinished),
+                            Err(error) => namui::event::send(Event::Error(error.to_string())),
+                        }
+                    });
```

Comment:
Can #438 be applied to it?

Reply:
That's not `must have to` thing. callback is just option of coding.